compiler/parser/Parser.y

```diff
@@ -1366,7 +1366,7 @@ interface_body_decl
   :
   --
   'choice' tycon OF_TYPE btype_ arecord_with_opt { sL (comb2 $2 $>) $ InterfaceChoiceSignature $ ChoiceSignature Nothing $2 $4 $5 }
-  | consuming_ 'choice' tycon OF_TYPE btype_ arecord_with_opt { sL (comb2 $3 $>) $ InterfaceChoiceSignature $ ChoiceSignature Nothing $3 $5 $6 }
+  | consuming 'choice' tycon OF_TYPE btype_ arecord_with_opt { sL (comb2 $3 $>) $ InterfaceChoiceSignature $ ChoiceSignature (unLoc $1) $3 $5 $6 }
```

Comment:
I don’t understand the change here. We have the case without a consuming declaration covered in the line above. Why do we need to go from `consuming_` to `consuming`? That looks like a parser ambiguity.

Reply:
that's right, it should be `consuming_`. Fixed it.